apps/hash-graph/bin/cli/src/subcommand.rs

```diff
@@ -33,3 +38,28 @@ pub enum Subcommand {
     #[cfg(all(hash_graph_test_environment, feature = "test-server"))]
     TestServer(TestServerArgs),
 }
+
+fn spawn_runtime(future: impl Future<Output = Result<(), GraphError>>) -> Result<(), GraphError> {
```

Comment:
Minor suggestion: Rename to `block_on`

Reply:
renamed in [6e591ab](https://github.com/hashintel/hash/pull/3091/commits/6e591ab5bb600a6b7881131b437fb0e044584f83)